src/greyVest/utils.js

```diff
@@ -0,0 +1,8 @@
+import { lensify } from '../utils/react'
+import F from 'futil-js'
+
+let binding = (...lens) => ({
```

Comment:
we should call this `open` or `openBinding` - a `binding` (as we've defined it) is a function that takes a lens and returns an object